moon-store-mac-os/moon-store-mac-os/Networking/NetworkManager.swift

```diff
@@ -175,4 +175,18 @@ final class NetworkManager {
             throw MSError.networkConnection
         }
     }
+    
+    // MARK: - Convert To dictionary
+    
+    func convertToDictionary<T: Codable>(data: T) throws -> [String: Any] {
```

Comment:
Por convencion cuando se transforma se usa la palabra `as`, mas contexto: https://github.com/Rexmoon/moon-store/pull/46/files#r1908129111

```suggestion
    func asDictionary<T: Codable>(from user: T) throws -> [String: Any] {
```